### PR TITLE
Post Settings: Status

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR is a part of changes to the Post Settings. The goal for the PR is just to remove the previous spinner and use the new dialog to update the status. The way the status is saved & retrieved is not the best right now, however, I am still in the process of figuring out how we want to save things locally and remotely and I'd rather make this change along with that. So long story short, please treat this PR as an intermediary change. Having said that, I am open to suggestions to better handle them, even if it won't happen in this PR.

To test:
* Go into Post Settings & change the status to play with different scenarios. (Notice that you can set a published post the Draft and go back to the post list, what should happen then and many more questions like this will be handled in the future PRs)

/cc @tonyr59h & @maxme (whoever can get to it sooner)

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java [hunk 1]
```diff
@@ -1024,6 +1011,27 @@ private void showTagsActivity() {
...
+    private void showStatusDialog() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setTitle(R.string.post_settings_status);
+        int checkedItem = getCurrentPostStatusIndex();
+        // Current index should never be -1, but if if is, we don't want to crash
+        if (checkedItem == -1) {
+            checkedItem = 0;
+        }
+        builder.setSingleChoiceItems(R.array.post_settings_statuses, checkedItem, null);
+        builder.setPositiveButton(R.string.dialog_button_ok, new DialogInterface.OnClickListener() {
```
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java]: We [recently started](https://github.com/wordpress-mobile/WordPress-Android/pull/5651) using `android.R.string.ok`, can you switch it here?

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java [hunk 2]
```diff
@@ -1024,6 +1011,27 @@ private void showTagsActivity() {
...
+            public void onClick(DialogInterface dialog, int which) {
+                ListView lw = ((AlertDialog)dialog).getListView();
```
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java]: Is this supposed to be `lv`? If not, just curious what `lw` means 😄

**Comment by oguzkocer** [comment]
Thanks for the review @tonyr59h! Fixed both issues and should be ready for another pass!

**Comment by tonyr59h** [comment]
Looks good! :shipit:

**Pull request merged**